website/content/docs/concepts/plugins/storage/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+page_title: Storage Plugins
+description: Learn how Nomad manages dynamic storage plugins.
+---
+
+# Storage plugins
+
+Nomad has built-in support for scheduling compute resources such as
+CPU, memory, and networking. Nomad's storage plugin support extends
+this to allow scheduling tasks with dynamically managed storage volumes.
+
+Choose between two types of storage plugins:
```

Comment:
We should probably link to https://developer.hashicorp.com/nomad/docs/operations/stateful-workloads from here but we should also add a section to that doc for DHV. We can follow-up with that in a separate PR though.